resolutions/-resolution-annual-xxx.md

```diff
@@ -0,0 +1,32 @@
+# ANNUAL RESOLUTIONS OF THE BOARD OF DIRECTORS
+
+**Resolution(s) in writing** of the board of directors of Hypha Worker Co-operative Inc. (the "Co-operative") 
+from the meeting of the directors held by electronic means on this [XXth day of Month, 20XX].
+
+**BACKGROUND:**
+
+1.  ...
+
+
+**IT WAS RESOLVED THAT:**
+
+1.  ...
+
+2.  ...
+
+
+Dated in the Province of Ontario on the [XXth day of Month, 20XX].
+
+__________________________________ :lock_with_ink_pen: [digital signature][sigfile-[director]]\
+[Director Name]
+
+__________________________________ :lock_with_ink_pen: [digital signature][sigfile-[director]]\
+[Director Name]
+
+__________________________________ :lock_with_ink_pen: [digital signature][sigfile-[director]]\
+[Director Name]
```

Comment:
Since we require only two signatures, I think we should only have two in the template.

```suggestion
```

Reply:
Sure, tho I think having total signatures for a batch of dec'ns is cool

Reply:
Yes just for purpose of the template we should put 2 as 3 seems arbitrary